Packs/Flashpoint/Integrations/Ignite/Ignite.py

```diff
@@ -142,7 +142,7 @@
 ALERT_RESOURCE_URL = {
     'communities': '/search/context/communities/{}',
     'marketplaces': '/search/context/marketplaces/{}',
-    'media': '/search/results/media?include.media_id={}',
+    'media': '/search/results/media?include.date=all+time&include.media_id={}',
```

Comment:
why did you changed the the original url?

Reply:
Hi @tcarmeli1, The old `media` based URL was not having the parameter to search it on bases of all time. So, we have added date parameter to resolve the issue.

Reply:
So it will not lead to backwards compatibility right?

Reply:
The change doesn't affect any functionality. Its only related to the platform side evaluation of the parameters.